src/actions/rootRequestedThunk.tsx

```diff
@@ -23,15 +23,12 @@ export const rootRequestedThunk = () => {
         const handleTheData = (dbrecords: any) => {
             const convert = (dbrecord: IDatabaseRecord) => {
                 return {
-                    parent:         -1,
+                    parent:         dbrecord.childof,
```

Comment:
The idea behind the -1 here was to use nonsense for the root, so that it would be easy to detect if it tried to ask for the parent of the root later on.